src/components/document-viewer/doc-fragment.jsx

```diff
@@ -412,7 +412,7 @@ class DocFragment extends Component {
             </Flex>
           )}
           <h2>
-            <AnchorLink text={this.props.content[this.getTitleKey()]} />
+            <AnchorLink text={this.props.content.getUuid()} />
```

Comment:
I guess something similar also ought to be done on line 429?

Reply:
good spot!